developer_tools/pylint_list

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+SCRIPT_DIRNAME=`dirname $0`
+SCRIPT_DIR=`(cd $SCRIPT_DIRNAME; pwd)`
+
+cd $SCRIPT_DIR/..
+
+#This is intended to be a useful list for things to fix, but cannot be
+# mandatory because they are too common, or are occasionally needed.
+
+RGX="[a-zA-Z_]{1,2}[a-zA-Z0-9]{0,50}$"
+METHOD_RGX="[a-zA-Z_]{1,2}[a-zA-Z0-9]{2,50}(__)?$"
+
+pylint --indent-string="  " --variable-rgx="$RGX" --attr-rgx="$RGX" --function-rgx="$RGX" --method-rgx="$METHOD_RGX" --const-rgx="$RGX" --argument-rgx="$RGX" --disable=bad-whitespace --disable=line-too-long --disable=multiple-statements --disable=multiple-imports --disable=import-error --disable=wrong-import-position --disable=unused-import --disable=wrong-import-order --disable=attribute-defined-outside-init --disable=too-many-branches --disable=too-many-locals --disable=too-many-statements --disable=too-many-nested-blocks --disable=too-many-instance-attributes --enable=missing-docstring --enable=no-absolute-import --enable=old-division --enable=print-statement --enable=file-builtin framework/
```

Comment:
Hm, as we fix #42 it probably will start giving better results.  I think most of the pylint import checks failed badly because of the way we were importing things.  I'll add a comment to that issue that we should look at adding more pylint checks when we are done fixing it.